### DISCUSSION TITLE
The models directory includes two types of pretrained models:

### DISCUSSION THREAD

**Discussion by yosiasz** [explosion/spaCy#6608] [Language Support]
Greetings,

Is this some stale entry in https://spacy.io/models 

The models directory includes two types of pretrained models:

because I do not see such a directory neither under spacy nor spacy-models.

Thanks

**Comment by svlandeg** [comment]
See the left column of that page: it has a section on "core models" and on "starter models".

**Reply by yosiasz** [reply]
@svlandeg wanted to add Amharic to spacy/models but other than creating a PR that will build it and see whether it passes or fails, is there any documentation on how to test locally the changes one makes ?
